content/scripts/build.js

```diff
@@ -978,63 +1117,39 @@ class Builder {
     return locales;
   }
 
-  excludeSlug(metadata) {
+  excludeSlug(url) {
```

Comment:
I don't like that. The URL is `/en-US/docs/Foo/Bar` but `metadata.slug` is `Foo/Bar` so if you do an CLI flag to filter only on those called "doc" you only want it to match the against "Foo/Bar". 
If I'm totally wrong, perhaps we should change the meaning of that CLI flag to be about filtering on URL instead.

Reply:
I didn't change the original functionality (which used the `mdn_url`) other than fixing it (we had changed the metadata such that it no longer contains `mdn_url`, so this hasn't worked since that change). However, I agree with your point that we should really be using the slug here rather than the url, so I'll change that!

Reply:
We'll make a mental note to fix that later then. I rarely use the slug filtering. But I often use the `-f` or `--foldersearch` so I only have to wait for a smaller handful of docs.